src/EventReporter/ConsoleSummaryEventReporter.php

```diff
@@ -106,7 +100,7 @@ private function printSummary(array $events, OutputInterface $output): void
     /**
      * @psalm-param list<EventInterface> $events
      * @psalm-param list<class-string> $desiredEvents
-     * @param EventInterface[] $events
+     * @var EventInterface[] $events
```

Comment:
It should be `@param` already after rebase, I wrote a blog post about rebasing conflicting branch https://samsonasik.wordpress.com/2015/09/16/practical-git-4-rebasing-conflicted-task-branch-against-primary-branch/ just in case you need it.